pkg/teleporter/relayer.go

```diff
@@ -247,7 +258,26 @@ func executeRelayer(binPath string, configPath string, logFile string) (int, err
 		return 0, err
 	}
 
-	return cmd.Process.Pid, nil
+	ch := make(chan struct{})
```

Comment:
we should move it into separate func in utils

Reply:
also belonging to a package in SDK, to be meged later on